README.md

```diff
@@ -1,4 +1,4 @@
-# GPT-Engineer
+# Ollama-Multi-threaded-UI-Integration for GPT-Engineer 
```

Comment:
changed from @6rz6 [Ollama_Models_Arena.py](https://github.com/gpt-engineer-org/gpt-engineer/pull/1098/files#diff-28d7c8481b3d39311b7059628a1d9020b6d8a6e95e87e0815ac24976d4c6cc31) 

Integrated on v.0.28->0.29

Reply:
Your changing the title of the readme?